From wfp5p@virginia.edu Mon Mar 16 17:32:45 2009
From: Bill Pemberton <wfp5p@virginia.edu>
Date: Mon, 16 Mar 2009 22:18:00 -0400
Subject: Staging: comedi: Remove comment mentioning typedefs
To: greg@kroah.com
Message-ID: <20090317021759.14467.72065.stgit@viridian.itc.Virginia.EDU>


There are no typedefs left in this file, so no need for a comment
pointing out the typedefs.

Signed-off-by: Bill Pemberton <wfp5p@virginia.edu>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---

 drivers/staging/comedi/drivers/dt2801.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/staging/comedi/drivers/dt2801.c
+++ b/drivers/staging/comedi/drivers/dt2801.c
@@ -43,7 +43,7 @@ Configuration options:
 /* Ports */
 #define DT2801_IOSIZE 2
 
-/* define's & typedef's */
+/* define's */
 /* ====================== */
 
 /* Commands */
